packages/synapse-react-client/src/mocks/msw/handlers/accessRequirementHandlers.ts

```diff
@@ -84,43 +90,110 @@ export const getAccessRequirementEntityBindingHandlers = (
   ),
 ]
 
-export const getAccessRequirementStatusHandlers = (
+export const getAccessRequirementsBoundToTeamHandler = (
   backendOrigin: string,
-  arStatus: Record<string, AccessRequirementStatus> = {},
-) => [
+  accessRequirements: AccessRequirement[] = [mockSelfSignAccessRequirement],
+) =>
   rest.get(
-    `${backendOrigin}${ACCESS_REQUIREMENT_STATUS(':id')}`,
+    `${backendOrigin}/repo/v1/team/:teamId/accessRequirement`,
 
     async (req, res, ctx) => {
-      let response: AccessRequirementStatus | undefined
-      const accessRequirement = mockAccessRequirements.find(
-        accessRequirement => req.params.id === accessRequirement.id.toString(),
-      )
-      if (typeof req.params.id === 'string' && req.params.id in arStatus) {
-        response = arStatus[req.params.id]
+      const status = 200
+      const response: PaginatedResults<AccessRequirement> = {
+        results: accessRequirements,
+        totalNumberOfResults: accessRequirements.length,
       }
-      if (!response && accessRequirement) {
-        const isManagedACTAR =
-          accessRequirement.concreteType ===
-          MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
-        response = {
-          accessRequirementId: req.params.id as string,
-          concreteType: isManagedACTAR
-            ? 'org.sagebionetworks.repo.model.dataaccess.ManagedACTAccessRequirementStatus'
-            : 'org.sagebionetworks.repo.model.dataaccess.BasicAccessRequirementStatus',
-          isApproved: true,
-          currentSubmissionStatus: isManagedACTAR
-            ? {
-                submissionId: mockApprovedSubmission.id,
-                submittedBy: mockApprovedSubmission.submittedBy,
-                modifiedOn: mockApprovedSubmission.modifiedOn,
-                state: SubmissionState.APPROVED,
-              }
-            : undefined,
+      return res(ctx.status(status), ctx.json(response))
+    },
+  )
+
+export const getAccessRequirementStatusHandlers = (
```

Comment:
Update this handler to reference the map variable to handle dynamically-changing AR statuses